setup.cfg

```diff
@@ -86,9 +86,9 @@ marrow/package = py.typed
 [options.entry_points]
 
 marrow.package.sample =
-	name = marrow.package:name
-	load = marrow.package:load
-	traverse = marrow.package:traverse
+	name = marrow.package.canonical:name
+	load = marrow.package.loader:load
+	traverse = marrow.package.loader:traverse
```

Comment:
This is a potentially worthy note to document: the plugin registration itself must match the dot-colon import path resolved by `name()`. If an object is aliased into place through import (e.g. promotion to the package scope), resolution will fail.